marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -380,12 +379,13 @@ public void clearUserModules() {
                     }
                 }
             );
-
+            //The custom mapping functions, ootb rest extensions and search options are not deleted by this method
```

Comment:
Sorry for being stubborn, but why are we not deleting these?

Reply:
Hitting "REDEPLOY" doesn't load these and they will be lost unless users run mlDeploy again

Reply:
One thing I don't understand is why LoadUserModulesCommand which is called when DataHubService.installUserModules() is run, doesn't load the custom mapping functions ? Could this also be the issue that @bsrikan mentioned in the curation slack room ?